.gitignore

```diff
@@ -0,0 +1,18 @@
+.idea
+venv/
+*/.DS_Store
+*.pyc
+app/*/__pycache__/
+.DS_Store
+#__init__.cpython-36.pyc
+#views.cpython-36.pyc
+#app/migrations/versions/*
+#/__pycache__/
+#*.cpython-36.pyc
```

Comment:
all your .pyc files could be caught with **/*.pyc ..same with your __pycache__ files => **/__pycache__/
try understand regexp..just the basics, it will be a huge help in the long run

Reply:
done, thanx